Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/MathGenerics.h

```diff
@@ -28,7 +28,7 @@ namespace ScriptCanvas
             return AZStd::stof(stringValue);
         }
 
-        SCRIPT_CANVAS_GENERIC_FUNCTION_NODE(StringToNumber, k_categoryName, "{FD2D9758-5EA2-45A3-B293-A748D951C4A3}", "Converts the given string to it's numeric representation if possible.", "", "");
+        SCRIPT_CANVAS_GENERIC_FUNCTION_NODE(StringToNumber, k_categoryName, "{FD2D9758-5EA2-45A3-B293-A748D951C4A3}", "Converts the given string to it's numeric representation if possible.", "StringValue");
```

Comment:
Can you provide before&after screenshot to show what has been changed?

And why do you need this change here? I think previously two quotes are for output and input?

Reply:
Sure, I can provide the screenshots for this node with/without this change. Based on the definition for other nodes using the SCRIPT_CANVAS_GENERIC_FUNCTION_NODE, the quotes after node description should be the node inputs.

#define SCRIPT_CANVAS_GENERIC_FUNCTION_NODE(NODE_NAME, CATEGORY, UUID, DESCRIPTION, ...)\
    SCRIPT_CANVAS_GENERIC_FUNCTION_MULTI_RESULTS_NODE_WITH_DEFAULTS(NODE_NAME, ScriptCanvas::NoDefaultArguments, CATEGORY, UUID, false, DESCRIPTION, __VA_ARGS__)

Reply:
actually the last part of arguments are for inputs and outputs name together, so do you really need this change, seems like you just need to add input name translation

Reply:
confirmed locally the issue is caused by trying to add slot with empty name
https://github.com/o3de/o3de/blob/development/Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Node.cpp#L2287

Fix is to give slot name, translation change is optional but not harm to have